src/shared/api/generated/CBioPortalAPI-docs.json

```diff
@@ -4488,10 +4488,6 @@
         "hugoGeneSymbol": {
           "type": "string"
         },
-        "length": {
```

Comment:
This length?

Reply:
yes length is removed: https://github.com/cBioPortal/cbioportal/pull/6312